benchmarks/ALCE/ELI5/README.md

```diff
@@ -0,0 +1,104 @@
+            "extraction": "Venture capitalists invest in startups with the expectation of a big payout down the road, either through an initial public offering or an acquisition. This means that the investors expect the company to eventually become profitable and generate returns on their investment. However, in the case of firms like Snapchat and Uber, they are valued highly because of their potential for future growth and dominance in their respective markets, rather than current profitability. Therefore, venture capitalists may not expect repayment within a specific number of years but instead expect a significant"
+        },
+        {
+            "title": "What Are Venture Capital Investments? \u2013 DollarsAndSense.my",
+            "text": "Ever wondered how highly valued technology giants like Google and Facebook were able to grow so fast and pay their employees so well in such a short amount of time, or how still growing start-ups like Uber are able to lose 1.2 billion US dollars in just the first half of this year alone and still command a valuation upwards of 50 billion US dollars? The answer lies with a special category of investment activity known as venture capital. Venture capitalists are professional investors who invest in a number of highly scalable high-risk technology ventures hoping to make a multi-fold",
+            "url": "http://dollarsandsense.my/what-are-venture-capital-investments/",
+            "summary": "Venture capitalists invest in highly scalable high-risk technology ventures, such as Snapchat and Uber, hoping to make a multi-fold return on their investment. This explains how firms can be valued highly despite not making a profit.",
+            "extraction": "The reason why firms like Uber can command such high valuations despite not making a profit is due to venture capital investments. Venture capitalists are professional investors who invest in high-risk technology ventures with the hopes of making a multi-fold return on their investment. Therefore, they do expect some form of repayment within a number of years but are willing to take on the risk in exchange for the potential high returns."
+        },
+        ...
+    ]
+}
+```
+
+## Metrics
+
+In the original paper, the authors evaluated four metrics on the ELI5 dataset: MAUVE, Claim Recall, Citation Recall, and Citation Precision. 
+
+Additionally, metrics such as ROUGE were also included in the evaluation, but no performance comparisons were made.
+
+### MAUVE
```

Comment:
You can just link the metric to rageval, and you don't need to explain the metrics again.